spec/unit/middleware/requeue_exceptions_spec.rb

```diff
@@ -16,8 +16,8 @@ def around_perform(job)
         end
       end
 
-      let(:container) do 
-        container = container_class.new 
+      let(:container) do
+        container = container_class.new
```

Comment:
I'm glad to see whitespace cleaned up, but in the future I would prefer to see whitespace cleanups in their own commit so they are not mixed with non-cosmetic code changes.